README.rst

```diff
@@ -40,34 +40,41 @@ You can install "pytest-splunk-addon" via `pip`_ from `PyPI`_::
 Developing
 ------------
 
-Note: Must install docker desktop.
+Note: Must setup kubernetes cluster [ Reference: `minikube`_ , `microk8s`_, `k3s`_ ], vscode or pycharm pro optional
+
+Note2: Appinspect requires libmagic verify this has been installed correctly each time a new workstation/vm is used https://dev.splunk.com/enterprise/docs/releaseapps/appinspect/splunkappinspectclitool/installappinspect
 
 .. code:: bash
 
     $ git clone --recurse-submodules -j8 git@github.com:splunk/pytest-splunk-addon.git
-    $ cd pytest-splunk-addon
-    $ poetry install
-    $ ... (change something)
-    # run unit tests
-    $ poetry run pytest tests/unit
-    # run some of the docker-based tests to verify end-to-end behaviour, example:
-    $ poetry run pytest -v --splunk-version=8.2 -m docker tests/test_splunk_addon.py::test_splunk_app_requirements_modinput
+    $ #setup python venv must be 3.7    
+    $ python3 -m venv .venv
 
+    $ source .venv/bin/activate
 
-Usage
------
+    $ cd pytest-splunk-addon
```

Comment:
This seems to be a section dedicated to how to start contributing to PSA, this seems to be showing how to activate a virtual environment.

Reply:
Updated the README.